dev-haskell/patch/patch-0.0.3.2.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
```

Comment:
Add a line here like `#hackport: flags: +split-these` so that this USE flag hard-enable is carried through version bumps automatically.